### PR TITLE
JBEAP-24958 Removing Reset test for Configuration => Interface

### PR DESCRIPTION
**Pull request by mchoma**

### CODE REVIEW

**Comment by mchoma** [comment]
@spriadka @kstekovi Please review

**Comment by mchoma** [comment]
Local run with changes
```
  (Results)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Tests:        1                                                                                │
  │ Passing:      1                                                                                │
  │ Failing:      0                                                                                │
  │ Pending:      0                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  0                                                                                │
  │ Video:        false                                                                            │
  │ Duration:     12 seconds                                                                       │
  │ Spec Ran:     test-configuration-interface.cy.ts                                               │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘


====================================================================================================

  (Run Finished)


       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  test-configuration-interface.cy.ts       00:12        1        1        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        00:12        1        1        -        -        -  

```

File: packages/testsuite/cypress/e2e/interface/test-configuration-interface.cy.ts [hunk 1]
```diff
@@ -44,9 +44,4 @@ describe("TESTS: Configuration => Interface", () => {
       cy.verifyAttribute(managementEndpoint, address, anyAddress, !value);
     });
   });
-
-  it("Reset", () => {
-    cy.navigateTo(managementEndpoint, "interface;name=" + interfaceToEdit.name);
-    cy.resetForm(configurationFormId, managementEndpoint, address);
```
Comment (mchoma) [inline packages/testsuite/cypress/e2e/interface/test-configuration-interface.cy.ts:50]: Alternatively this line can be written to expect current warning, but don't know how to write that.

**Comment by mchoma** [comment]
> Can you update the description of the PR and commit

Done

File: packages/testsuite/cypress/e2e/interface/test-configuration-interface.cy.ts (hunk 1 above)
Comment (OndrejKotek) [reply packages/testsuite/cypress/e2e/interface/test-configuration-interface.cy.ts:50]: Let's remove the test for now. We'll update this method in scope of #44.

**Pull request merged**